app/models/comment.rb

```diff
@@ -1,2 +1,12 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
```

Comment:
[OPTIONAL] Kindly remove the class_name present here because the purpose is to allow the naming of a class to be different from what Rails expects. And rails already knows that model names are to be singular